src/Microsoft.CodeQuality.Analyzers/UnitTests/Performance/DoNotUseCountWhenAnyCanBeUsedTests.cs

```diff
@@ -0,0 +1,550 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Operations;
+using Microsoft.CodeAnalysis.Testing;
+using Microsoft.CodeAnalysis.VisualBasic;
+using Microsoft.CodeQuality.CSharp.Analyzers.Maintainability;
+using Microsoft.CodeQuality.VisualBasic.Analyzers.Maintainability;
+using Test.Utilities;
+using Xunit;
+using static Microsoft.CodeQuality.Analyzers.Performance.UnitTests.DoNotUseCountWhenAnyCanBeUsedTestData;
+
+namespace Microsoft.CodeQuality.Analyzers.Performance.UnitTests
+{
+
+    public class DoNotUseCountWhenAnyCanBeUsedAnalyzerTests : DiagnosticAnalyzerTestBase
```

Comment:
💡 `DiagnosticAnalyzerTestBase` is the legacy testing harness. New tests should use `VerifyCS` and `VerifyVB`.